lib/matplotlib/patches.py

```diff
@@ -1053,23 +1064,26 @@ def _update_path(self):
             self._path = Path(np.vstack(verts), np.hstack(codes))
 
     def get_data(self):
-        """Get `.StepPatch` values and edges."""
-        return self._values, self._edges
+        """Get `.StepPatch` values, edges and baseline."""
+        return self._values, self._edges, self._baseline
```

Comment:
For better usability:

```suggestion
        StairData = namedtuple('StairData', 'values, edges, basline')
        return StairData(self._values, self._edges, self._baseline)
```
and adapt the documentation.